### ISSUE TITLE
Unresponsive & Misplaced UI Elements

### ISSUE THREAD

**Issue by ZethiaZ** [GrapheneOS/os-issue-tracker#3474]
Over the last few months I have been noticing an ongoing issue with UI elements becoming unresponsive or being entirely misplaced.

Examples include:
- Autorotate and the Bluetooth toggle within the drawer accessed by swiping down from the top becoming unresponsive.
- App closing/opening screen (accesed by swiping up from the bottom) freezing when trying to close apps and requires toggling the phone on/off (but not fully restarting to resolve temporarily)
- Gesture navigation bar becoming stuck to the wrong side of the phone. (Occurs only after frozen autorotate)

Photo Examples:
![Screenshot_20240427-031032](https://github.com/GrapheneOS/os-issue-tracker/assets/48283890/fd67dab6-3f15-49de-b719-e98a44fa17b6)
![Screenshot_20240427-031830](https://github.com/GrapheneOS/os-issue-tracker/assets/48283890/97be42e7-f7ff-45dd-8ae9-fbfd858b1725)
(I will try my best to get more videos and photos of these occuring)

How to reproduce:
I'm not sure exactly what's triggering these issues, but they tend to occur more frequently as my phone has been on longer and occur under my normal use. Rebooting always resolves the issues.

Other info:
I am using a Pixel 8 on GrapheneOS with the build number 2024042200.

Other comments: 
I will try to get more videos and pictures, and note any other related issues, I believe I am forgetting some. If this is a repeat with another issue please let me know, I couldn't find a identical issue already open. If there are any logs that could be helpful please let me know! In my very non expert opinion this sounds like some kind of memory leak and all these issues seem related, but if I need to split them up as well then please tell me.
